src/pip/_internal/commands/list.py

```diff
@@ -213,14 +213,20 @@ def latest_info(dist):
             # This is done for 2x speed up of requests to pypi.org
             # so that "real time" of this function
             # is almost equal to "user time"
```

Comment:
This comment should likely be rewritten.

Reply:
... or just moved inside the "try" block.